draft-ietf-tls-rfc8446bis.md

```diff
@@ -3517,6 +3517,16 @@ a KeyUpdate with the old key is received before accepting any messages
 encrypted with the new key. Failure to do so may allow message truncation
 attacks.
 
+With a 128-bit key as in AES-128, rekeying 2^64 times has a high
+probability of key reuse within a given connection.  Note that even
+if the key repeats, the IV is also independently generated.  In order
+to provide an extra margin of security, sending implementations MUST
+NOT allow the epoch to exceed 2^48-1.  In order to allow this value
+to be changed later, receiving implementations MUST NOT enforce this
+rule.  If a sending implementation receives a KeyUpdate with
```

Comment:
Well, this is all based on a predicate (128-bit keys) that might not hold true for all cipher suites.  I would instead say that this version of the specification applies this limit for all AEADs, no matter what their key size, but implementations MUST NOT enforce this limit as it could be extended in future versions of the specification.